pants

```diff
@@ -193,4 +205,16 @@ pants_dir="$(bootstrap_pants "${pants_version}" "${python}")"
 # See https://blog.phusion.nl/2017/10/13/why-ruby-app-servers-break-on-macos-high-sierra-and-what-can-be-done-about-it/
 export no_proxy='*'
 
-exec "${pants_dir}/bin/python" "${pants_dir}/bin/pants" "$@"
+if [[ -z "${pants_config_file}" ]]; then
+  pants_config_file_flag="";
+else
+  pants_config_file_flag=--pants-config-files="[\"${PWD}/${pants_config_file}\"]";
+fi
+
+# MacOS's default open files limit is too low for Pants, bump it to something reasonable.
+if [[ "$OSTYPE" == "darwin"* ]]; then
+  ulimit -n 10000
+fi
```

Comment:
If possible, it'd be better to save this for a followup. This is the type of change that could cause surprising behavior for users so it would help to have a distinct commit for things like `git bisect`.

Reply:
Another reason this would be good to do in a followup. If the user already has configured this value to > 10,000, then we should respect what they used.

Reply:
Done.